test/iterable_basics_test.dart

```diff
@@ -199,219 +199,193 @@ void main() {
       final strings = <String>['a', 'aaa', 'aa'];
 
       expect(
-        strings.max((a, b) => a.length.compareTo(b.length)).value,
-        equals('aaa'),
-      );
+          strings.max((a, b) => a.length.compareTo(b.length))!, equals('aaa'));
```

Comment:
nit: personally I prefer how the formatter formats this when there is a trailing comma.
```suggestion
          strings.max((a, b) => a.length.compareTo(b.length))!, equals('aaa'),);
```

Reply:
![image](https://user-images.githubusercontent.com/15900150/96916290-5b466000-1475-11eb-8f55-d2943e0e20e8.png) 😂